### DISCUSSION TITLE
Clicking on the trigger element on mobile briefly hides then re-opens tooltip

### DISCUSSION THREAD

**Discussion by BerciTheBeast** [Akryum/v-tooltip#668] [Q&A]
**Problem:** When clicking on the trigger element on mobile while the tooltip is already open the tooltip fades out a bit then pops right back up.

**Expected behavior:** Clicking on the trigger element would toggle the tooltip, like on desktop

**Link to reproduction:**
- [here](https://jsbin.com/dudowecido/edit?html,css,js,output)
- also in plaintext for the paranoid folk: [https://jsbin.com/dudowecido/edit?html,css,js,output](https://jsbin.com/dudowecido/edit?html,css,js,output)

![vue-tooltip-bug](https://user-images.githubusercontent.com/16097715/81191913-ddbb9000-8fb9-11ea-8865-5b3176ee3181.gif)

**Additional notes:** on iPhones, Safari, it sometimes happens that the first press of the trigger element only briefly flashes the tooltip and it has to be pressed again for it to open. One would expect that the tooltip would open when the trigger element is clicked.

EDIT: It appears that most of the time on Safari mobile, the tooltip just doesn't appear the first time you press the trigger.

**Comment by dorogoff** [comment]
I have the same issue if using ```trigger: "hover click".```

**Comment by Kinzi** [comment]
This solved the issues on mobile for me:

```
Vue.use(VTooltip, {
  defaultTrigger: window.innerWidth > 768 ? 'hover focus click' : 'click'
})
```

**Comment by dorogoff** [comment]
> This solved the issues on mobile for me:
> 
> ```
> Vue.use(VTooltip, {
>   defaultTrigger: window.innerWidth > 768 ? 'hover focus click' : 'click'
> })
> ```

Good idea, but I found the other way, using a **```<button>```** or **```<a>```** element, then it works well for me.
https://jsbin.com/riyiqacuge/2/edit?html,css,js,output

**Comment by existe-deja** [comment]
> This solved the issues on mobile for me:
> 
> ```
> Vue.use(VTooltip, {
>   defaultTrigger: window.innerWidth > 768 ? 'hover focus click' : 'click'
> })
> ```

It doesn't work with ssr. Setting `defaultTrigger: 'hover click'` fixed it for me.
